2.6_testers2.ipynb

```diff
@@ -247,10 +299,19 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "In this section we will look at running sections of a unit test in parallel.\n",
-    "Look at the use of `fork` in the example below.\n",
-    "The chain `fork` methods allows these blocks of code to be executed at the same time, independent of each other.\n",
-    "The `join` method tells the test to wait until both `fork` blocks have completed before moving on to whatever comes next.\n"
+    "In this section we will look at running sections of a unit test concurrently. In order to do this we will introduce 4 new features of testers2.\n",
+    "\n",
+    "| method | description |\n",
+    "| :---   | :---        |\n",
+    "| fork   | launches a concurrent code block, additional forks can be run concurrently to this one via the .fork appended to end of the code block of the preceeding fork |\n",
+    "| join | re-unites multiple related forks back into a single thread |\n",
+    "| enqueueSeq | Continues to add (enqueue) elements from the `Seq` to a `Decoupled` input interface, one at a time, until the sequence is exhausted |\n",
```

Comment:
Conceptually I'd separate these from fork and join, they're also not concurrent operations.

Reply:
Done